Documentation/reference/config.md

```diff
@@ -513,11 +513,11 @@ a string value
 A shared key distributed between all parties signing and verifying JWTs.
 ```
 
-#### &emsp;&emsp;issuer: ""
+#### &emsp;&emsp;issuer: []string
 ```
-a string value
+a list of string value
 
-The "Issuer" key is what the service expects to verify as the "issuer" claim.
+A list of issuers to verify. An empty list will accept any issuer in a jwt claim.
```

Comment:
I think it should fail-closed. The relevant section of config can be commented out if jwt auth isn't desired.

Reply:
Per the spec, iss optional:

```
  The "iss" (issuer) claim identifies the principal that issued the
   JWT.  The processing of this claim is generally application specific.
   The "iss" value is a case-sensitive string containing a StringOrURI
   value.  Use of this claim is OPTIONAL.
```

there can be use cases where a client may have ephemeral services coming up and down, if they create their own issuer with a hostname or some other ephemeral node value, forcing an issue would break this situation.